### PR TITLE
cmd/bosun: Show description metadata on graph page

### PR DESCRIPTION
**Pull request by kylebrandt**
Adds support to the MetadataMetrics API for specifying the metric

### CODE REVIEW

**Comment by kylebrandt** [comment]
image

image

File: cmd/bosun/web/static/js/graph.ts [hunk 1]
```diff
@@ -384,6 +398,7 @@ bosunControllers.controller('GraphCtrl', ['$scope', '$http', '$location', '$rout
 					graphRefresh = $timeout(() => { get(true); }, 5000);
 				};
 			});
+
```
Comment (mjibson) [inline cmd/bosun/web/static/js/graph.ts]: boo

File: cmd/bosun/web/static/js/graph.ts [hunk 2]
```diff
@@ -344,16 +345,29 @@ bosunControllers.controller('GraphCtrl', ['$scope', '$http', '$location', '$rout
 		return;
 	}
 	var autods = $scope.autods ? '&autods=' + $('#chart').width() : '';
+	function getMetricMeta(metric: string) {
+		$http.get('/api/metadata/metrics?' + 'metric=' + metric)
```
Comment (mjibson) [inline cmd/bosun/web/static/js/graph.ts]: `'/api/metadata/metrics?metric=' + getURIComponent(metric)

Comment (kylebrandt) [inline cmd/bosun/web/static/js/graph.ts]: @mjibson encodeURIComponent() ?

Comment (mjibson) [inline cmd/bosun/web/static/js/graph.ts]: yeah

**Pull request merged**